src/site/content/en/design-system/pattern-library.md

```diff
@@ -0,0 +1,23 @@
+---
+title: 'Pattern Library'
+permalink: '/design-system/pattern-library/index.html'
+layout: 'design-system-documentation.njk'
+---
+
+## Todo
+
+- Flesh out more information about patterns
+
+## Includes
+
+Including partials within partials works exactly the same as within Eleventy templates. The include base is set to `src/site/_includes`.
+
+For example, if you want to include an icon from `src/site/_includes/icon`, you would write the following.
```

Comment:
I think we may want to tell folks to use our svg or icon macros. They take additional arguments if you want to apply a class or aria label to the SVG

Reply:
Ah yeh I'll update all that